docs/getting-started/data-views-in-sec.asciidoc

```diff
@@ -0,0 +1,33 @@
+[[data-views-in-sec]]
+= {Data-sources} in Elastic Security
+
+The active {data-source} determines which data appears throughout {es-sec}.
+{Data-sources} are defined by the index patterns they include.
+Only data from the {es} {ref}/documents-indices.html[indices], {ref}/data-streams.html[data streams], or {ref}/alias.html[index aliases] specified in the active {data-source} will appear.
+
+
+== How to switch {data-sources}
+
+You can tell which {data-source} is active by clicking the {Data-source} button that appears at the upper right of {es-sec} pages that display data, such as Overview, Alerts, or Hosts.
+To switch to another existing {data-source}, click the same button {Data-source} and choose a different {data-source} under `Choose {data-source}`:
+
+image::images/dataview-default-setting-highlighted.png[the securitySolution:defaultIndex setting, shown in context]
+
+
+== How to create or modify {data-sources}:
+
+To modify or delete an existing {data-source} or to define a new one, you need the required permissions.
+To learn more, refer to {apm-app-ref}/data-views.html[{kib} {Data-Sources}].
+
+
+== The default {data-source}
+
+The first time you add data to an {es-sec} instance within a given {apm-app-ref}/xpack-spaces.html[space], the default {data-source} gets generated for that space and becomes active.
+If you delete the active {data-source} when there are no other defined {data-sources}, the default {data-source} will regenerate and become active upon the refresh of any {es-sec} page in that space, because the app requires a {data-source} to function.
```

Comment:
What do you think about putting this in an admonition to emphasize it more? I feel like this is something we want to call out, especially for users who are working with custom indices and data views.

Reply:
I think you're absolutely right that we should highlight this info for people working with custom indices. I added an admonition after the intro paragraph instead of in the "Default data view" section so it would be more prominent. What do you think?